openedx/core/djangoapps/credit/views.py

```diff
@@ -29,6 +29,59 @@
 log = logging.getLogger(__name__)
 
 
+@require_GET
+def get_provider_detail(request, provider_id):
+    """
+    get the details of required provider
+
+    This end-point will get the details of providers against
+    given parameter provider_id.
+
+    **Example Usage:**
+
+        Get /api/credit/v1/provider/hogwarts
+
+        Response: 200 OK
+        Content-Type: application/json
+        {
+            "url": "http://localhost:8000/api/credit/v1/provider/info/hogwarts",
+            "method": "GET",
+            "response": {
+                "provider_id": "hogwarts",
```

Comment:
Why are many of these prefixed with provider? Given that this response is returned by the provider detail endpoint, it doesn't make much sense to add the prefix.